dockers/docker-sonic-gnmi/base_image_files/monit_gnmi.j2

```diff
@@ -1,9 +1,9 @@
 ###############################################################################
 ## Monit configuration for telemetry container
 ###############################################################################
-{% if SMARTSWITCH is defined and SMARTSWITCH == 1 %}
+{%- if SMARTSWITCH is defined and SMARTSWITCH == "1" %}
```

Comment:
>SMARTSWITCH [](http://example.com/codeflow?start=34&length=11)

Could you add a testcase to prevent regression?

Reply:
Sure, I will add end to end test

Reply:
I think a unit test for j2 render is just enough.